FeedAPIChallenge/Feed API/FeedItemsMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedItemsMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Oleksii Levchenko on 13.09.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal enum FeedItemsMapper {
```

Comment:
You can consider removing `internal` here, as it's default access control. It's explicitly written in the course only for teaching purposes.
```suggestion
enum FeedItemsMapper {
```